📖Use nice tip syntax for callouts

tharders opened the pull request:
To highlight text blocks as callouts on amp.dev currently a complex template code is used: `{% call callout('Read on', type='read') %}`
amp.dev also supports a nice tag syntax that will produce the same callouts: `[tip type="read-on"]`
All existing callouts where converted to the much nicer syntax.

Please note that the extra text in the callout call ('Read on' in the example above) is not used on amp.dev and also not supported by the tip tag and therefore removed.
Compare https://amp.dev/documentation/components/amp-form?format=websites and https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md

The tip tag supports the following types (the types are converted accordingly)
 * note
 * read-on
 * important
 * success

tharders committed:
Convert ampbyexample iframes to inline preview

tharders committed:
In markdown use nice tip syntax for callouts

tharders committed:
In markdown improve formatting of nice tip syntax for callouts

mrjoro commented:
/cc @wg-outreach

CrystalOnScript reviewed (changes requested):
Thanks so much or doing this! I flagged the tip types we no longer support. Unfortunately I had to read what was in them and most of them reference ampbyexample - hoping it isn't too much trouble to update those.

mrjoro commented:
Hello; is this PR still active?

tharders committed:
Update master with upstream changes

tharders committed:
Merge remote-tracking branch 'origin/master' into feature/use_nice_tip_tags_for_callouts

# Conflicts:
#	builtins/amp-img.md
#	extensions/amp-accordion/amp-accordion.md
#	extensions/amp-ad/amp-ad.md
#	extensions/amp-analytics/amp-analytics.md
#	extensions/amp-bind/amp-bind.md
#	extensions/amp-call-tracking/amp-call-tracking.md
#	extensions/amp-form/amp-form.md
#	extensions/amp-iframe/amp-iframe.md
#	extensions/amp-lightbox/amp-lightbox.md
#	extensions/amp-list/amp-list.md
#	extensions/amp-next-page/amp-next-page.md
#	extensions/amp-script/amp-script.md
#	extensions/amp-selector/amp-selector.md
#	extensions/amp-sidebar/0.2/amp-sidebar.md.draft
#	extensions/amp-story/amp-story.md
#	extensions/amp-user-notification/amp-user-notification.md
#	spec/amp-actions-and-events.md
#	spec/amp-cors-requests.md

tharders committed:
Replace callout with tips in latest documentation

After feedback from CrystalOnScript:
also change the tips link texts where "AMP By Example" was mentioned
and change 'success' tips to 'note' or 'read-on'

tharders committed:
Change last "success" tip to "note"

tharders commented:
I'm very sorry, I somehow missed the feedback here.
In the meantime lots of changes where made, but some `callout` still remained which I have changed to tips.
I have also changed the `success` tips according to the feedback from @CrystalOnScript and also changed the link texts where "AMP By Example" was mentioned.
The links itself where all changed before and already point to the amp.dev site.

tharders committed:
Merge remote-tracking branch 'upstream/master'

tharders committed:
Merge remote-tracking branch 'origin/master' into feature/use_nice_tip_tags_for_callouts

# Conflicts:
#	extensions/amp-ad-exit/amp-ad-exit.md

kristoferbaxter commented:
@CrystalOnScript is the PR still active?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ampproject/amphtml?pullRequest=23028) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ampproject/amphtml?pullRequest=23028) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ampproject/amphtml?pullRequest=23028) it.</sub>
